scripts/install.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+set -e
```

Comment:
can we please remove that :-(
it is nice for debugging, but it shouldn't be in final script

Reply:
ah, sorry, my fault. I thought this is `set -x` 😬